### PR TITLE
POST received events to status_url in channel status worker

### PR DESCRIPTION
**Pull request by justinvdm**
#44 adds a worker that consumes status events published by a transport. The plan is to change the worker to post received events to a `status_url` given to it via its config.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
So, the `assert_was_logged` appears to have been giving us false positives, fixing, not ready for review yet.

File: junebug/tests/helpers.py [hunk 1]
```diff
@@ -161,3 +201,28 @@ def get_dispatched_messages(self, queue):
...
+
+        if body is not None:
+            self.assertEqual(json.loads(req['body']), body)
+
+    def assert_body_contains(self, req, **fields):
+        body = json.loads(req['body'])
+
+        self.assertEqual(
+            dict((k, v) for k, v in body.iteritems()),
+            body)
```
Comment (rudigiesler) [inline junebug/tests/helpers.py]: I'm a little confused as to what this is actually doing. Is it not just comparing `body` to itself? Should we be comparing it to `fields`?

**Comment by justinvdm** [comment]
Ok, ready for review again.

File: junebug/tests/helpers.py (hunk 1 above)
Comment (justinvdm) [inline junebug/tests/helpers.py]: Ya, this looks wrong, should be comparing to fields, thanks, fixing.

Comment (justinvdm) [inline junebug/tests/helpers.py]: Done.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**